<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Reporte_Panel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Buscar</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1191</width>
       <height>421</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1189</width>
        <height>419</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1191</width>
         <height>421</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLineEdit" name="buscar_usuario_admin_txt_area">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="buscar_usuario_admin_btn">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_orden_tabla_usuario">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>20</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>PreOrder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InOrder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PostOrder</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="aplicar_orden_comboBox_orden_tabla_usuario">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Aplicar</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tabla_buscar_admin">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>1151</width>
          <height>341</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>191</number>
        </attribute>
        <column>
         <property name="text">
          <string>Nombres</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Apellidos</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Correo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nacimiento</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Orden</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Carga Masiva</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="Usuarios_boton_archivo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>133</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar archivo</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="Solicitudes_boton_archivo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar archivo</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>70</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="Publicaciones_boton_archivo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar archivo</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Usuarios</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Solicitudes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>40</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Publicaciones</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Reporte</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1191</width>
       <height>431</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1189</width>
        <height>429</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>551</width>
         <height>351</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="arbol_usuario_frame" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>351</height>
         </rect>
        </property>
        <widget class="QLabel" name="arbol_usuario_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>541</width>
           <height>351</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>551</width>
         <height>351</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="lista_doble_publis_frame" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>351</height>
         </rect>
        </property>
        <widget class="QLabel" name="lista_doble_publis_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>541</width>
           <height>351</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Arbol de usuarios</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>0</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Lista de publicaciones</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Generar_reporte_btn">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>390</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Generar reporte</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Informacion Admin</string>
    </attribute>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>501</width>
       <height>311</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ESTRUCTURA DE DATOS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FACULTAD :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CIENCIAS Y SISTEMAS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>APELLIDOS :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ALVAREZ HERRERA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>KEVIN ANDRES</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CARNET :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NOMBRES :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>202203038</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="CerrarSesion_boton_2">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar sesion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
